examples/postgresql/connect.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# Create a directory to store the certificate, key and bundle
+declare -r SVIDS_DIR=./svids
+mkdir -p $SVIDS_DIR
+
+# Fetch SVIDs from SPIRE agent
+/opt/spire/spire-agent api fetch -write ${SVIDS_DIR}
+
+# Change key permissions (required by psql)
+chmod 600 $SVIDS_DIR/svid.0.key
+
+# Connecto to postgres using the certificates fetched
+psql "port=5432 host=localhost user=postgres-user dbname=testdb sslcert=$SVIDS_DIR/svid.0.pem sslkey=$SVIDS_DIR/svid.0.key sslrootcert=$SVIDS_DIR/bundle.0.pem sslmode=verify-ca"
```

Comment:
maybe you can mention all those parameters for sslcert are not neccesary in case they use default path for them?
https://www.postgresql.org/docs/12/libpq-ssl.html#LIBPQ-SSL-FILE-USAGE